builtin/ffi_overrides/init.lua

```diff
@@ -0,0 +1,66 @@
+local ie = ...
+
+if not _G.core.settings:get_bool("secure.use_ffi", true) then
```

Comment:
why the _G?

Reply:
This was a suggestion by @Desour. Access to the global environment should be explicit to avoid security loopholes.

Reply:
I'd be interested to hear about that but in any case it doesn't matter for the init stuff.
Aside: If this is important for security it must be written down in the file!

Reply:
Indeed not important for init in builtin. But if you just call a global function (from `_G`) after a mod is run, that function can be overridden. Therefore, it's imo better to have to explicitly write `_G` in that cases. This is checked via luacheck (and therefore `_G` is also needed here).
Anyone with proper lua knowledge should be aware about the security risk described above, therefore I don't think writing this down is needed (but wouldn't hurt either).

Reply:
I added a comment to `builtin/ffi_overrides/init.lua`.